region/crateria/east.json

```diff
@@ -1482,6 +1482,25 @@
                   ],
                   "devNote": "Could use a failure definition?"
                 },
+                {
+                  "name": "West Ocean Bug Ice Clip",
+                  "notable": true,
+                  "requires": [
+                    "canNonTrivialIceClip"
+                  ],
+                  "note": [
+                    "Freeze a bug a three tiles to the right of the morph tunnel entrance, directly under the tile where ceiling is higher.",
+                    "Spin jump onto the bug, crouch, and jump up.",
+                    "With HiJump it is more lenient, otherwise the bug must be at an exact pixel height.",
+                    "If you have XRay and Morph, it can be done more easily:",
+                    "unequip HiJump (if you have it) and freeze the bug high enough that you can roll under it, unmorph, jump to clip up through it;",
+                    "aim down as you jump up through the bug, then morph & unmorph, perform an XRay forced stand up, and jump.",
```

Comment:
I'm not sure what "aim down as you jump up through the bug" means.  
Is it to avoid a jump morph?
To get on the bug, an aim down jump seems more niche and not really required compared to jump morph.
And positioning for the clip is done before the jump to the bug and after the morph.